docs/build-repro.md

```diff
@@ -65,9 +65,12 @@ See also the [documentation](/frontend/dockerfile/docs/reference.md#buildkit-bui
 
 ## Caveats
```

Comment:
nit: The limitation described in this PR should be documented somewhere?
> Limitations:
>
>    containerd 1.7 is needed for the containerd worker mode. (https://github.com/containerd/containerd/commit/9c9f564a35df7990870a53a125afbf88ac412753)

Reply:
Mentioned in `docs/build-repro.md`